File: App/src/main/java/com/example/mchapagai/activity/BaseActivity.java
package com.example.mchapagai.activity;

import android.os.Bundle;

import dagger.android.support.DaggerAppCompatActivity;

public abstract class BaseActivity extends DaggerAppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public void setContentView(int resId) {
        super.setContentView(resId);
    }

}

File: App/src/main/java/com/example/mchapagai/base/BaseActivity.kt
package com.example.mchapagai.base

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

abstract class BaseActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun setContentView(resId: Int) {
        super.setContentView(resId)
    }
}